s3_management/manage.py

```diff
@@ -7,6 +7,7 @@
 from collections import defaultdict
 from typing import List, Type, Dict, Set, TypeVar, Generator, Optional
 from re import sub, match
+from packaging.version import parse
```

Comment:
Is this from a third party library?

Reply:
It's https://github.com/pypa/packaging (which is dependency for wheel)

Reply:
Yes, I have added the dependency to requirements.txt. Thanks for pointing it out.